spec/features/collection_drafts/forms/spatial_information_form_spec.rb

```diff
@@ -103,6 +103,19 @@
               fill_in 'Maximum Value', with: '15.0'
             end
           end
+          click_on 'Add another Tiling Identification System'
+
+          within '.multiple-item-2' do
```

Comment:
I am of two minds.  

1. I don't think that we need to test filling in a specific enum when we change the schema. To that end, I think it would be sufficient to check that the list of choices is correct.
2. The entire point of this schema change is to add this enum, so testing that the schema change was successful means publishing this value.

Reply:
I don't know that we need a specific new test based on this change. However, I am a little surprised that we don't have a test for the form that this enum appears on, where we test that all the options appear in the select. If that does not exist we should create such a test.